static/script/application.js

```diff
@@ -93,16 +93,27 @@ require.def('antie/application',
                         }
                     }
 
+                    var syncStartUp = function() {
+                        self.run();
+                        self.route(device.getCurrentRoute());
+                    };
+
                     require([_layout.module], function (layout) {
-                        self.setLayout(layout, styleBaseUrl, imageBaseUrl, _layout.css, _layout.classes, [], function () {
-                            self.run();
-                            self.route(device.getCurrentRoute());
+                        self.setLayout(layout, styleBaseUrl, imageBaseUrl, _layout.css, _layout.classes, [], function(){
```

Comment:
Yeh its interesting because setLayout itself is in some way async already (when it gets passed a callback) 

So those startup calls where already being handled asynchronously in some situations.

At least that was my understanding from reading the code.

An alternative to this might have been to wrap this https://github.com/fmtvp/tal/blob/d7273966258f0f43a2c73e8af80e6157a8e7bdc9/static/script/application.js#L196 in a setTimeout.

I don't have clue which is 'better'. Its all pretty complicated.